src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -251,8 +251,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Bubble this up, so our new control knows how big we want the font.
         _FontSizeChangedHandlers(actualNewSize.width, actualNewSize.height, true);
 
-        // Turn the rendering back on now that we're ready to go.
-        _renderer->EnablePainting();

```

Comment:
this scares me. we disabled painting while tearing out, so why would we not reenable it?

Reply:
As Mike noted, it seems like the call was redundant and so I'll keep this change in if you don't mind. I've updated the PR message though.

Reply:
sh-...should we leave a comment explaining that?